### PR TITLE
ANGOLASUP-516: restore 'Remove the No Batch Defined options'

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openlmis.atlassian.net/browse/ANGOLASUP-516

I restored changes that were overwritten in previous release.

### CODE REVIEW

File: src/stock-orderable-group/orderable-group.service.js [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+            } else {
+                lots = _.chain(orderableGroup).pluck('lot')
+                    .compact()
+                    .value();
+
+                lots.forEach(function(lot) {
+                    lot.expirationDate = dateUtils.toDate(lot.expirationDate);
+                });
+
+                // ANGOLASUP-516: Removed the 'No Lot Defined' option
```
Comment (NikolaLaskowska) [inline src/stock-orderable-group/orderable-group.service.js:79]: please add another comment which indicates where the customization ends, ex ANGOLASUP-516: ends here

File: src/stock-orderable-group/orderable-group.service.spec.js [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    });
+
+    it('should find lots in orderable group', function() {
+        //given
+        var group = [this.item1, this.item2];
+
+        //when
+        var lots = service.lotsOf(group);
+
+        // ANGOLASUP-516: Removed the 'No Lot Defined' option
```
Comment (NikolaLaskowska) [inline src/stock-orderable-group/orderable-group.service.spec.js:141]: same here

File: src/stock-orderable-group/orderable-group.service.js (hunk 1 above)
Comment (sniedzielski) [reply src/stock-orderable-group/orderable-group.service.js:79]: It is related to the 'removed' code, not 'added' one (as we discussed on slack)

File: src/stock-orderable-group/orderable-group.service.spec.js (hunk 2 above)
Comment (sniedzielski) [reply src/stock-orderable-group/orderable-group.service.spec.js:141]: It is related to the 'removed' code, not 'added' one (as we discussed on slack)

**Pull request merged**